### ISSUE TITLE
Pirates need to be able to access their bar vend

### ISSUE THREAD

**Issue by Lyroth001** [Pixeltheaertist/Trieste-Port-14#117]
## Description
How else can they get piss drunk?

**Comment by Lyroth001** [comment]
They should also have a radio, maybe freebooter?

**Comment by Pixeltheaertist** [comment]
yeah agreed, the ones yesterday had their outpost handmade and just put the spawns down manually. THEY MUST BE ABLE TO GET WILDLY DRUNK!! UNACCEPTABLE!

**Comment by Pixeltheaertist** [comment]
closing this mainly because the one in referenced round was impromptu spawned
